_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
+| [Kubernetes: Service Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)               | Beta
 | [Pluggable Key/Cert Support for Istio CA](https://istio.io/docs/tasks/security/plugin-ca-cert.html)        | Beta
-| [Istio on Istio (secure istio components)](https://docs.google.com/document/d/1YzYPddihbLgJhme27-md9COn0NnKUKX_xCJ_GPXU4Fw/edit#heading=h.jbmfrt4h5lj0) | Alpha
-| [Incremental mTLS](https://docs.google.com/document/d/1D7wZCQjVB72Wlwr5ZxP5WUmn3FUDr-XzfX8OodPXe8Y/edit)    | Alpha
-| [VM Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)         | Alpha
+| [Service-to-service mutual TLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
+| [Incremental Enablement of service-to-service mutual TLS](https://docs.google.com/document/d/1D7wZCQjVB72Wlwr5ZxP5WUmn3FUDr-XzfX8OodPXe8Y/edit)    | Alpha
+| [VM: Service Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)         | Alpha
+| [OPA Checker](https://github.com/mangchiandjjoe/istio/blob/d5390f6e436225949907d77ad3e9747a9bc26722/mixer/adapter/opa/README.md)  - **New to 0.4**      | Alpha
 
-### User Interface
 
 
-| Feature           | Phase       
-|-------------------|-------------------
-| [Grafana Dashboard](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html) | Alpha
-| [Service Graph](https://istio.io/docs/tasks/telemetry/servicegraph.html)    | Alpha
+### Core
 
 
-### Performance and Scalability
+| Feature           | Phase        
+|-------------------|-------------------
+| [Kubernetes: Istio Data Plane: Envoy Installation and Traffic Interception](https://istio.io/docs/setup/kubernetes/)        | Beta
+| [Kubernetes: Istio Control Plane Installation](https://istio.io/docs/setup/kubernetes/) | Beta
+| [Pilot Integration into Kubernetes Service Discovery](https://istio.io/docs/setup/kubernetes/)         | Beta
+| [Attribute Expression Language](https://istio.io/docs/reference/config/mixer/expression-language.html)        | Beta
+| [Mixer Adapter Authoring Model](https://istio.io/blog/posts/2017/adapter-model.html)        | Beta
+| [VM: Istio Data Plane: Envoy Installation, Traffic Interception and Service Registration](https://istio.io/docs/guides/integrating-vms.html)    | Alpha
+| [VM: Istio Control Plane: Installation and Upgrade (Galley, Mixer, Pilot, CA)](https://github.com/istio/istio/issues/2083)  | Alpha
+| [Kubernetes: Istio Control Plane Upgrade](https://istio.io/docs/setup/kubernetes/) | Alpha
+| [Pilot Integration into Consul, Eureka and Cloud Foundry Service Discovery](https://istio.io/docs/setup/consul/quick-start.html)     		   | Alpha
+| [Basic Config Resource Validation](https://github.com/istio/istio/issues/1894)         	   | Alpha
+| [Distributed Tracing to Zipkin / Jaeger](https://istio.io/docs/tasks/telemetry/distributed-tracing.html)         | Alpha
```

Comment:
Move to observability.. and its beta btw.

Reply:
Contemplated back and forth on whether this should be alpha or beta and decided to take a conservative approach. 
About the tracks, Sven suggested it should be part of core than observability, so I'll leave that up to him.

Reply:
Tracing seems fine as observability, but keep alpha until we've reviewed it to make sure it meets beta criteria.